deployment/start_and_deploy_to_server.sh

```diff
@@ -50,22 +50,11 @@ scp -r "$construction_dir" ubuntu@"$target_server_url":"$location"
 ssh ubuntu@"$target_server_url" "mv \"$keycloak_backup_dir\"/*-users-*.json \"$keycloak_user_dir\" || true"
 
 echo "Set up Keycloak from scratch."
-ssh ubuntu@"$target_server_url" "export KEYCLOAK_UPLOADER_VALUE=\"$KEYCLOAK_UPLOADER_VALUE\";
-                                 export KEYCLOAK_UPLOADER_SALT=\"$KEYCLOAK_UPLOADER_SALT\";
-                                 export KEYCLOAK_READER_VALUE=\"$KEYCLOAK_READER_VALUE\";
-                                 export KEYCLOAK_READER_SALT=\"$KEYCLOAK_READER_SALT\";
-                                 export KEYCLOAK_ADMIN=\"$KEYCLOAK_ADMIN\";
-                                 export KEYCLOAK_ADMIN_PASSWORD=\"$KEYCLOAK_ADMIN_PASSWORD\";
-                                 export KEYCLOAK_GOOGLE_SECRET=\"$KEYCLOAK_GOOGLE_SECRET\";
-                                 export KEYCLOAK_GOOGLE_ID=\"$KEYCLOAK_GOOGLE_ID\";
-                                 export KEYCLOAK_LINKEDIN_ID=\"$KEYCLOAK_LINKEDIN_ID\";
-                                 export KEYCLOAK_LINKEDIN_SECRET=\"$KEYCLOAK_LINKEDIN_SECRET\";
-                                 export KEYCLOAK_MAILJET_API_SECRET=\"$KEYCLOAK_MAILJET_API_SECRET\";
-                                 export KEYCLOAK_MAILJET_API_ID=\"$KEYCLOAK_MAILJET_API_ID\";
+ssh ubuntu@"$target_server_url" "set -o allexport; source \"$location\"/.env; set +o allexport;
```

Comment:
Wollen wir das wirklich im Klartext als Datei auf dem Server haben?

Reply:
Ich glaube das ist nicht gefährlicher, als es im Klartext in Docker-Container im Env zu haben, oder?

Reply:
das file auf dem server kann jeder lesen, im container sollte eine Stufe restriktiver sein

Reply:
im prinzip ja - aber auf unseren Servern gibt es nur den ubuntu-user - und der ist sudoer.